### PR TITLE
Added a Log Level to Keep the Logger Quiet

### PR DESCRIPTION
**Pull request by joemccann**
Added a property to the Tesseract object and just a quick fix to keep the logger quiet.  The `process` method's signature has changed to pass in the log level as the 2nd argument.

### CODE REVIEW

File: lib/nodecr.js [hunk 1]
```diff
@@ -57,6 +62,9 @@ var tesseract = {
   },
 
   _runTesseract: function(image, callback, languageCode, pageSegMode, config) {
+    
+    tesseract.logLevel
```
Comment (joscha) [inline lib/nodecr.js:66]: What is this supposed to be doing?

File: lib/nodecr.js [hunk 2]
```diff
@@ -163,6 +172,7 @@ var ConvertPreprocessor = function(inputFile, callback) {
 
 // Exports
 
+module.exports.logOutput = tesseract.logOutput;
```
Comment (joscha) [inline lib/nodecr.js:175]: What does this do?

**Comment by joscha** [comment]
Hi Joe, thank you for your contribution - that's actually quite useful when processing lots of files.

Instead of the flag, how about exposing a log function pointer that by default logs to console.log but can be overridden with any implementation the user wants. That would not only be more flexible but also avoid an additional flag that isn't boolean but would need further development to extend more option values.

**Comment by joemccann** [comment]
Indeed, but that's why I just applied a quick patch to quiet the logging that was in the code.  If you feel that's how you want this lib to exist, that's totally fine, but I'm just quickly quieting the logging.  

## 

Joe McCann
Creative Technologist

subPrint Interactive
Fon:      +1.718.577.2343  
Fax:      +1.718.218.3499
Email:   joe@subprint.com (mailto:joe@subprint.com)
Web:     http://subprint.com (http://subprint.com/)

**Comment by joscha** [comment]
I agree with you it should have that option, however I'd prefer a more flexible approach - I think about merging this in - maybe in a slight variation of what you provided. Thank you for making me aware though!

**Comment by joscha** [comment]
I added this option in 0.0.5

**Pull request closed without merging**